lib/strings.md

```diff
@@ -5,20 +5,21 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A string is a data type that consists of a combination of characters or values. Strings are surrounded by "" or '' marks. 
 
 # What are some examples of information that would be Strings as opposed to some other data type?
 
-Replace me with your response...
+Letters, words, sentences, or numbers that are read or printed to the screen and are not telling the computer to do something would be examples of strings. Numbers which are used mathematically and boolean operators like true or false would not be strings.
 
 # What is one way, using Ruby, to retrieve the 6th character in a String like `"Ada Lovelace"`? How about the 8th character? What happens if you try to retrieve the value of the _99th_ character (Or any character that doesn't exist)?
 
-Replace me with your response...
+A string is an array, so each value in a string has an index number, starting at 0. For example, since it is the 5th value, the index for "L" in "Ada Lovelace" is 4. So to retrieve the 6th character, I would look for the value with an index of 5. The code "Ada Lovelace" [5] would return "o". For the 8th character, I would look for the value with an index of 7. If I search for an index that is greater than the number of characters in a string, it returns nil.
```

Comment:
Strings _are_ indexed in the same way as Arrays but they are not Arrays, themselves. Otherwise, a great explanation.